q2_vsearch/plugin_setup.py

```diff
@@ -282,7 +282,7 @@
         'minlen': qiime2.plugin.Int % qiime2.plugin.Range(0, None),
         'maxns': qiime2.plugin.Int % qiime2.plugin.Range(0, None),
         'allowmergestagger': qiime2.plugin.Bool,
-        'minovlen': qiime2.plugin.Int % qiime2.plugin.Range(0, None),
+        'minovlen': qiime2.plugin.Int % qiime2.plugin.Range(5, None),
```

Comment:
Why this change? I'm guessing vsearch enforces this so you're bumping it up?

Reply:
yep!